<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCamConfig</class>
 <widget class="QWidget" name="FormCamConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Setting</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>image sizeï¼š</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxCamSize"/>
     </item>
     <item>
      <widget class="QPushButton" name="pb_set_config_image_size">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>write</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>exposure            </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_exposure">
       <property name="text">
        <string>auto exposure</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_exposure">
       <property name="text">
        <string>5000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_set_config_exposure">
       <property name="text">
        <string>write</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>fps:                                  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_fps">
       <property name="text">
        <string>30</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_set_config_fps">
       <property name="text">
        <string>write</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="cb_syncmode">
       <property name="text">
        <string>sync mode      </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_set_config_sync_mode">
       <property name="text">
        <string>write</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_set_config_sync_start">
       <property name="text">
        <string>start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_set_config_sync_stop">
       <property name="text">
        <string>stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>infrared pwm                          </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="hs_pwm">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_set_config_pwm">
       <property name="text">
        <string>write</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_set_config">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>config</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_exit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_cam_stop">
   <property name="geometry">
    <rect>
     <x>204</x>
     <y>310</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>close camera</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_cam_start">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>310</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>open camera</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_sync">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>sync</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
